Tensile/KernelWriterAssembly.py

```diff
@@ -3325,20 +3389,21 @@ def graWorkGroup(self, kernel, isPap):
         # re-mapping WG2 to WGKSerial -> wg2
         # re-mapping SerialWorkGroupIter to WGIJSerial -> wg0/1
         kStr += self.comment1("compute SerialWorkGroupIter / problemNumGroupTiles0x1 (aka nWG0*nWG1)")
-        kStr += self.sMagicDiv(kernel, stmp, sgpr("SerialWorkGroupIter"), sgpr("MagicNumProblemNumGroupTiles0By1"), 31)
+        kStr += self.sMagicDivAlg2(kernel, stmp, sgpr("SerialWorkGroupIter"), sgpr("MagicNumProblemNumGroupTiles0By1"), sgpr("MagicShiftProblemNumGroupTiles0By1"))
         kStr += inst("s_mov_b32", sgpr("WGKSerial"), sgpr(stmp), "wgKSerial = SerialWorkGroupIter / problemNumGroupTiles0x1")
-        kStr += inst("s_mul_i32", sgpr("WGIJSerial"), sgpr(stmp), sgpr("NumWorkGroups0By1"), "remainder part 1 : quotient * divisor")
+        kStr += inst("s_mul_i32", sgpr("WGIJSerial"), sgpr(stmp), sgpr("NumWorkGroups0"), "remainder part 1 : quotient * divisor (divisor = NumWorkGroups0 x 1)")
+        kStr += inst("s_mul_i32", sgpr("WGIJSerial"), sgpr(stmp), sgpr("NumWorkGroups1"), "remainder part 1 : quotient * divisor (divisor = NumWorkGroups0 x 1)")
```

Comment:
Typo in comment

Reply:
Thanks for spotting this. 
I also found that the old code is wrong for getting WGIJSerial = stmp * NumWorkGroups0 * NumWorkGroups0.